cypress/integration/test.js

```diff
@@ -2,6 +2,7 @@ describe("Testing Example App", function() {
   context("Initialising", function() {
     beforeEach(function() {
       cy.visit("http://localhost:8000/examples/index.html");
+      cy.get('[href="#Simple"]').click();
```

Comment:
@Danwhy the Cypress tests _appear_ to only cover the `#Simple` use-case.
Am I _missing_ the _range_ tests...?

Reply:
No, you're right; I should add some tests in to cover the range functionality. That example didn't exist when I wrote the tests, and I've yet to update them

Reply:
@Danwhy feel free to create a `new` issue for `DRP` tests 
(_as they aren't required for this Heroku PR to "pass" it's original acceptance criteria_) 

But IMO, the `DRP` is the most _useful_ use-case 
which is why AirBnB shows them _first_ in their example: http://airbnb.io/react-dates 😉